package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "malevich",
+  "version": "1.0.0",
+  "description": "TypeScript / React / HTML5 Training Exercise",
+  "main": "./dist/bundle.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/vlebedeff/malevich.git"
+  },
+  "author": "Vlad Lebedev, Vitalie Lazu",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/vlebedeff/malevich/issues"
+  },
+  "homepage": "https://github.com/vlebedeff/malevich#readme",
+  "dependencies": {
+    "postcss-import": "^8.1.2",
+    "react": "^15.2.1",
+    "react-dom": "^15.2.1",
+    "react-redux": "^4.4.5",
+    "redux": "^3.5.2"
+  },
+  "devDependencies": {
+    "css-loader": "^0.23.1",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "postcss": "^5.0.21",
+    "postcss-loader": "^0.9.1",
+    "postcss-nested": "^1.0.0",
+    "source-map-loader": "^0.1.5",
+    "style-loader": "^0.13.1",
+    "ts-loader": "^0.8.2",
```

Comment:
better to use a local dep. i tried building your project and it failed to compile because typescript wasn't found.